lib/exgboost.ex

```diff
@@ -88,30 +58,28 @@ defmodule EXGBoost do
   to `EXGBoost.train/2`. See the [XGBoost documentation](https://xgboost.readthedocs.io/en/latest/parameter.html) for a full
   list of parameters.
 
-
-  ```elixir
-  EXGBoost.train(X,
-                y,
-                obj: &EXGBoost.Training.train/1,
-                evals: [{X_test, y_test, "test"}],
-                learning_rates: fn i -> i/10 end,
-                num_boost_round: 10,
-                early_stopping_rounds: 3,
-                params: [max_depth: 3, eval_metric: ["rmse","logloss"]])
-  ```
+      EXGBoost.train(
+        x,
+        y,
+        obj: :multi_softprob,
+        evals: [{x_test, y_test, "test"}],
+        learning_rates: fn i -> i / 10 end,
+        num_boost_round: 10,
+        early_stopping_rounds: 3,
+        params: [max_depth: 3, eval_metric: ["rmse", "logloss"]]
```

Comment:
```suggestion
        max_depth: 3, 
        eval_metric: [:rmse, :logloss]
```

I forgot to update this example when the parameter API changed